docset/winserver2025-ps/smbshare/Unblock-SmbClientAccessToServer.md

```diff
@@ -0,0 +1,265 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: SmbClientAccessToServer.cdxml-help.xml
+Module Name: SmbShare
+ms.date: 02/22/2024
+online version: https://learn.microsoft.com/powershell/module/smbshare/unblock-smbclientaccesstoserver?view=windowsserver2025-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Unblock-SmbClientAccessToServer
+---
+
+# Unblock-SmbClientAccessToServer
+
+## SYNOPSIS
+Unblocks SMB client access to a previously blocked server.
+
+## SYNTAX
+
+```
+Unblock-SmbClientAccessToServer [-Name] <String[]> -IdentifierType <IdentifierType>
+ -Identifier <String> [-Force] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+```
+Unblock-SmbClientAccessToServer -InputObject <CimInstance[]> -IdentifierType <IdentifierType>
+ -Identifier <String> [-Force] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Unblock-SmbClientAccessToServer` cmdlet is used to allow SMB clients to access a server that
+was previously blocked using the `Block-SmbClientAccessToServer` cmdlet. When you run this cmdlet,
+any existing block on SMB client connections to the specified server will be lifted, and new
+connections will be allowed.
+
+## EXAMPLES
+
+### Example 1: Unblock SMB client access to a server
+
+```powershell
+$params = @{
+    Name = "Server01"
+    IdentifierType = "ISSUER"
+    Identifier = "CN=MyCertificateIssuer"
+    Force = $true
+}
+Unblock-SmbClientAccessToServer @params
+```
+
+This command unblocks SMB client access to a server named "Server01" for the based on the client's
+certificate. This command runs without prompting for confirmation as the `-Force` parameter is set
+to `$true`.
+
+## PARAMETERS
+
+### -AsJob
+
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to
+complete.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CimSession
+
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a session
+object, such as the output of a [New-CimSession](/powershell/module/cimcmdlets/new-cimsession) or
+[Get-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227966) cmdlet. The default is the
+current session on the local computer.
+
+```yaml
+Type: CimSession[]
+Parameter Sets: (All)
+Aliases: Session
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
```

Comment:
**Confirm** and **WhatIf** parameters go at the end of the list before CommonParameters.